modules/command/src/main/java/org/apache/fluo/command/FluoScan.java

```diff
@@ -79,7 +139,7 @@ public String getRowPrefix() {
 
     public ScanUtil.ScanOpts getScanOpts() {
       return new ScanUtil.ScanOpts(startRow, endRow, columns, exactRow, rowPrefix, help,
-          hexEncNonAscii, scanAccumuloTable);
+          hexEncNonAscii, scanAccumuloTable, exportAsCsv, exportAsJson);
     }
 
     public static ScanOptions parse(String[] args) {
```

Comment:
It would be nice to do some sanity checks and I think those could be done here.  Could check for the following.

 * Error if `--csv` and `--json` are both set.
 * Error if `--csv-some-option` is set and `--csv` is not set.
 * Error if the delimiter, escape, or quote is more than one char.

Reply:
Perfect! I'll do it.